Xkcd-Comics/ContentViews/ExplanationSheet.swift

```diff
@@ -7,12 +7,12 @@
 
 import SwiftUI
 
-struct ExplanationSheet: View {    
-    var explainationSheetVM = ExplanationSheetVM()
-    var num: Int
+struct ExplanationSheet: View {
+    let viewModel: ExplanationSheetVM
```

Comment:
Missing: `@ObservedObject`

ExplanationSheetVM is an ObservableObject, but without `@ObservedObject`, features such as `@Published` will not work. 

However, ExplanationSheetVM doesn't contain any mutable state, so you have two choices:

a) Remove `ObservableObject` inheritance from ExplanationSheetVM and let it be defined as a simple stateless class
b) Add `@ObservedObject` to the property here (if you think it will need to be mutable in the future)